apps/condo/domains/organization/utils/helpers.ts

```diff
@@ -27,11 +27,20 @@ export const roleToQuery = (rolesIds: Array<string>): OrganizationEmployeeRoleWh
     }
 }
 
-export const roleToSearchQuery = (roleName: string): OrganizationEmployeeRoleWhereInput => {
+export const roleToSearchQuery = (search: string, translates) => {
+    let whereIn = []
```

Comment:
this way it would be more readable and a bit more optimal
```
    const searchLCase = search.toLowerCase()
    const whereIn = !translates ? [] : Object.keys(translates).filter(
        key => (
            key.includes('employee.role')
            && !key.includes('description')
            && translates[key].toLowerCase().includes(searchLCase)
        )
    )
```